lab_1/main.py

```diff
@@ -12,7 +12,22 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+    if not isinstance(text, str):
+        return []
+    f = open('data.txt', encoding='utf8')
```

Comment:
You do not need to read from file in this function, `text` argument is of `str` format.

Also, do not forget to use `.close()` for files. Or open files with context manager `with open(file)`